bookchain-miner/src/http/dtos/peerdtos.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <oatpp/core/macro/codegen.hpp>
+#include <oatpp/parser/json/mapping/ObjectMapper.hpp>
+
+namespace bookchain::http {
+
+/* Begin DTO code-generation */
+#include OATPP_CODEGEN_BEGIN(DTO)
+
+class PeerLinkDto : public oatpp::data::mapping::type::Object {
```

Comment:
Can you explain the logic behind separating peers into 2 separate DTOs?

Reply:
One is for linking one is the actual data. So the PeerLinkDto will be used to syncing up /linking two nodes, whereas the PeerDto represents an actual peer object and is used to transmit peer data (for the whole peers list)